package.js

```diff
@@ -0,0 +1,19 @@
+var packageName = 'rubenv:angular-gettext';
+
+var where = 'client';
+var version = '2.1.0';
```

Comment:
I'd rather not update this every time we roll a release. Could we make this something like?

```js
var version = require('./package.json').version;
```